job_spec/INSAR_ISCE.yml

```diff
@@ -93,7 +93,9 @@ INSAR_ISCE:
       cost: 1.0
   validators: []
   compute_environment:
-    name: 'Default'
+    name: 'Aria'
+    allocation_type: EC2
+    allocation_strategy: BEST_FIT_PROGRESSIVE
```

Comment:
Compute environment is rendering like:
```yaml
  AriaComputeEnvironment:
    Type: AWS::Batch::ComputeEnvironment
    Properties:
      ServiceRole: !GetAtt BatchServiceRole.Arn
      Type: MANAGED
      ComputeResources:
        Type: EC2
        AllocationStrategy: BEST_FIT_PROGRESSIVE
        MinvCpus: 0
        MaxvCpus: !Ref MaxvCpus
        InstanceTypes:
          - !Ref InstanceTypes
```
but InstanceTypes should look like:
```yaml
        InstanceTypes: !Ref InstanceTypes
```

Reply:
Ah, InstanceTypes is being rendered like a list, but it should be a comma-deliminated string; fixed in https://github.com/ASFHyP3/hyp3/pull/2399/commits/70952d01e0232de66a0f2d4a9cafcee0c793dbbd